pkg/pb/contacts.override.gorm.go

```diff
@@ -30,3 +38,50 @@ func (m *ContactORM) AfterToPB(ctx context.Context, c *Contact) error {
 	}
 	return nil
 }
+
+// Hooks and direct db access:
+// Hooks Documentation: http://gorm.io/docs/hooks.html
+// Database access: http://gorm.io/docs/sql_builder.html
+// Below is an example that shows how to add a hook that runs before the Update method and how to access the database directly.
+
+// BeforeUpdate runs before the Update method and checks to see if the ID provided already exists.
```

Comment:
Please add a disclaimer that this is for demonstration purposes and that this particular check shouldn't be emulated :)